src/components/tooltip/tooltip.tsx

```diff
@@ -20,8 +20,9 @@ const DEFAULT_MAX_LENGTH = 50;
  *
  * :::note
  * In order to display the tooltip, the tooltip element and its trigger element
- * must be within the same document or document fragment.
- * A good practice is to just place them next to each other like below:
+ * must be within the same document or document fragment (the same shadow-root).
+ * Often, it's easiest to just place them next to each other like in the example
+ * below, but if you need to, you can place them differently.
```

Comment:
How about this change instead? @Kiarokh

Reply:
I changed from "shadow-root" to "shadowRoot", that's why the code commented now shows as outdated.

Reply:
Looks great to me! Squash?